packages/features/schedules/components/Schedule.tsx

```diff
@@ -46,7 +46,7 @@ const ScheduleDay = <TFieldValues extends FieldValues>({
   return (
     <div className="mb-1 flex w-full flex-col py-1 sm:flex-row">
       {/* Label & switch container */}
-      <div className="flex h-11 items-center justify-between">
+      <div className="flex h-11 w-32 items-center justify-between break-normal">
```

Comment:
@gitstart `break-normal` is the default if I'm correct, right? Does add anything here? 
Cool change apart from that, thanks a lot! 🙏

Reply:
Could you also make this full width on mobile, and only `w-32` starting at a bigger breakpoint? Makes it a bit nicer on mobile I think!